plugin/trino-hive/src/main/java/io/trino/plugin/hive/avro/AvroHiveFileUtils.java

```diff
@@ -67,7 +67,7 @@ public final class AvroHiveFileUtils
 {
     private final AtomicInteger recordNameSuffix = new AtomicInteger(0);
 
-    private AvroHiveFileUtils() {}
+    public AvroHiveFileUtils() {}
```

Comment:
Could you revert and change other methods static instead?

Reply:
done.